test/addons-napi/test_general/test.js

```diff
@@ -34,7 +34,7 @@ assert.notStrictEqual(test_general.testGetPrototype(baseObject),
 
 // test version management functions
 // expected version is currently 4
-assert.strictEqual(test_general.testGetVersion(), 6);
+assert.strictEqual(test_general.testGetVersion(), 7);
```

Comment:
I think the comment above should've also been changed but since this is a backport it's fine like this.